packages/terra-application-layout/docs/example.md

```diff
@@ -0,0 +1 @@
+# Terra Application Layout
```

Comment:
If we don't or won't have example docs, I say we remove this file.

Reply:
This is here solely to provide the same header for the Example page that About/Utils have too. I agree, its a little silly, but it looks more consistent on the site.